### PR TITLE
remove wrap-cli-callback

### PR DESCRIPTION
**Pull request by ricardo-devis-agullo**
From what I can see `wrap-cli-callback` was only used for CLI commands where we didn't pass a callback. We can just pass it explicitly on `cli/index.js` when calling yargs, and then we don't need to wrap every command. This also makes it easier to be able to [universalify](https://github.com/RyanZim/universalify) those functions (it would get confused thinking it's a promise if sometimes they get called without a callback).

### CODE REVIEW

File: src/cli/facade/clean.js [hunk 1]
```diff
@@ -3,7 +3,6 @@
 const read = require('read');
 
 const strings = require('../../resources/index');
-const wrapCliCallback = require('../wrap-cli-callback');
```
Comment (pbazydlo) [inline src/cli/facade/clean.js:6]: by doing this aren't we breaking contract for things like this:
https://github.com/opencomponents/grunt-oc/blob/master/tasks/oc-mock.js

Comment (ricardo-devis-agullo) [reply src/cli/facade/clean.js:6]: how? in that file it's explicitly passing a callback.

Comment (pbazydlo) [reply src/cli/facade/clean.js:6]: we've chatted about this, the difference is about programmatic api, however we think that the use-case of not passing a callback in programmatic api does not make any sense. @matteofigus we're thinking of treating this as a non breaking change - is that ok?

Comment (matteofigus) [reply src/cli/facade/clean.js:6]: yeah makes sense, thanks

**Pull request merged**